src/wp-a11y-day/style.css

```diff
@@ -1966,12 +1966,43 @@ input[type="submit"],button,.button {
 	padding:12px 24px
 }
 
-.wp-block-button:not(.is-style-button-underline) a:hover, .wp-block-button:not(.is-style-button-underline) a:focus,
+.wp-block-button:not(.is-style-button-underline, .is-style-arrow) a:hover, .wp-block-button:not(.is-style-button-underline, .is-style-arrow) a:focus,
 input[type="submit"]:hover,input[type="submit"]:focus,button:hover,button:focus,.button:hover,.button:focus {
 	background:var(--button-bg-hover);
 	color:var(--button-color-hover);
 }
 
+.wp-block-button.is-style-arrow .wp-block-button__link {
```

Comment:
I think this will need some styles for .is-style-arrow to make it display: block; and to fill its container (width: 100%). This style should never be used with multiple blocks; it's intended to be solitary. 

The container also needs to be styled...but that may just require us to add a class to the containing paragraph, to get the SVG background.

Reply:
Also needs to override border radius.

Reply:
@joedolson I didn't set the border-radius or width because they can be set in the block editor. 
![Screen Shot 2023-08-04 at 6 14 02 PM](https://github.com/wpa11yday/wpaccessibilityday/assets/835568/c88a3a25-ca44-458e-ab94-ae38d83832a0)
![Screen Shot 2023-08-04 at 6 14 12 PM](https://github.com/wpa11yday/wpaccessibilityday/assets/835568/f5a01f8d-8303-4175-bda8-13297446a806)

I think the way I'd build the container is a cover block and potentially use some columns blocks to align things.

That can all be stored as a block pattern to make it easy to reuse.

Reply:
I guess we can do it that way. As long as we don't have to set those values every time; we'll never want a border radius, and it should always be 100% width of it's parent container, minus padding. Works for me. Hey, I don't edit content on this site, anyway, so it's not my problem in the end. ;)